packages/near-membrane-base/src/environment.ts

```diff
@@ -128,6 +159,18 @@ export class VirtualEnvironment {
         }
     }
 
+    link(path: string) {
```

Comment:
All the changes in this file are about implementation of the linkage mechanism between the two sides. This method allows passing a path (assuming dot notation), and based on that, it will connect the object/function resolved using the provided path in both sides.